test/components/submission/filters.spec.js

```diff
@@ -229,7 +220,7 @@ describe('SubmissionFilters', () => {
           ]);
         })
         .beforeEachResponse((_, { url }) => {
-          const match = url.match(/&%24filter=__system%2FsubmissionDate\+ge\+([^+]+)\+and\+__system%2FsubmissionDate\+le\+([^+]+)(&|$)/);
+          const match = url.match(/&%24filter=__system%2FsubmissionDate\+ge\+([^+]+)\+and\+__system%2FsubmissionDate\+le\+([^&]+)(&|$)/);
```

Comment:
A small mistake in the regular expression. It only started being an issue now that there is a query parameter that can come after `$filter`. Instead of parsing the query string like this, we could use something like `relativeUrl(url).searchParams.get('$filter')`. However, I think that's something we can do in the future; I just wanted to make this small change for now.